app/src/main/java/com/zulip/android/networking/AsyncGetEvents.java

```diff
@@ -188,7 +188,11 @@ public Void call() throws Exception {
                 mActivity.runOnUiThread(new Runnable() {
                     @Override
                     public void run() {
-                        mActivity.getPeopleAdapter().refresh();
+                        try {
+                            mActivity.refreshPeopleDrawer();
```

Comment:
I maybe wrong but I think in order to solve the addressed issue we need to listen for "presence" type events which will give us the updates on the presence of realm users. I believe a lot of it is already implemented. See [presence wrapper](https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/networking/response/events/PresenceWrapper.java) and [process events](https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/networking/response/events/PresenceWrapper.java).